packages.json

```diff
@@ -26886,5 +26886,20 @@
     "description": "Ray tracing",
     "license": "GPL3",
     "web": "https://github.com/teob97/T-RayX"
+  },
+  {
+    "name": "util",
+    "url": "https://github.com/thisago/util",
+    "method": "git",
+    "tags": [
+      "html",
+      "utility",
+      "nim",
+      "nim-lang",
```

Comment:
```suggestion
```

Hi, the two tags don't seeem to be necessary.

Reply:
oh, this is because I copied project tags in github, i will fix